.changeset/pink-parents-help.md

```diff
@@ -0,0 +1,5 @@
+---
+"bigtest": patch
+---
+
+Modify post function in webdriver for firefox
```

Comment:
This is the how, not what/why. Let's add a refererence to the github issue that this fixes. Something like "fixes [url-to-issue] by adding content type to all webdriver requests"

Reply:
I haven't created an issue for it. Should I create the issue? Or just describe the what/why?

Reply:
Nah, this description is much better.